### DISCUSSION TITLE
Is the preset for nvidia H.265 preset-nvidia-hevc? [0.12.0-beta1]

### DISCUSSION THREAD

**Discussion by thinkloop** [blakeblackshear/frigate#4906] [Ask A Question] [answered]
The new docs mention that the preset for H.264 is: `preset-nvidia-h264`

Is the preset for H.265: `preset-nvidia-hevc`?

Thank you.

**Comment by NickM-27** [comment, marked as answer]
It's actually `preset-nvidia-h265`
